test_rmw_implementation/test/test_subscription.cpp

```diff
@@ -377,3 +377,40 @@ TEST_F(CLASSNAME(TestSubscriptionUse, RMW_IMPLEMENTATION), count_mismatched_subs
   EXPECT_EQ(RMW_RET_OK, ret) << rmw_get_error_string().str;
   EXPECT_EQ(0u, publisher_count);
 }
+
+TEST_F(CLASSNAME(TestSubscriptionUse, RMW_IMPLEMENTATION), take_serialized_with_bad_args) {
```

Comment:
@Lobotuerk this test case name is already used by #129.

Reply:
solved on cc0ea14